src/test/java/it/gov/pagopa/payment/test/fakers/SyncTrxStatusFaker.java

```diff
@@ -0,0 +1,22 @@
+package it.gov.pagopa.payment.test.fakers;
+
+import it.gov.pagopa.payment.dto.mapper.TransactionInProgress2SyncTrxStatusMapper;
+import it.gov.pagopa.payment.dto.qrcode.SyncTrxStatusDTO;
+import it.gov.pagopa.payment.enums.SyncTrxStatus;
+import it.gov.pagopa.payment.model.TransactionInProgress;
+import it.gov.pagopa.payment.utils.RewardConstants;
+
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+
+public class SyncTrxStatusFaker {
+    public static SyncTrxStatusDTO mockInstance(Integer bias){
```

Comment:
when building a useful mock instance builder we are interested to have the more generic (and valid) form of the object (which probably will be the more often used form).

Here you are returning a precise use case, where the trx has been rejected due to the unexistance of the initiatives...

Probably you have done this because you are testing this particular useCase, but this could not be the more generic "mockInstance" that someone could be interested to retrieve!

I would let the generic "mockInstance" to simply return the "map" of the TransactionInProgress "mockInstance"

If the useCase implemented here is frequent, let's implement it inside a "mockNoInitiativeInstance"
which would call the "SyncTrxStatusFaker .mockInstance" setting the fields required to change